app/src/main/java/com/example/gradeassist/RetrofitClientInstance.kt

```diff
@@ -1,28 +1,26 @@
 package com.example.gradeassist
 
-//import retrofit2.Retrofit
-//import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
 
-    // RetorofitClientInstance
+ private var retrofit: Retrofit? = null
+  private val BASE_URL = "https://pkgstore.datahub.io"
 
-//    private var retrofit: Retrofit? = null
-//    private val BASE_URL = "https://pkgstore.datahub.io"
-//
-//    val retrofitInstance: Retrofit?
-//        get() {
-//
-//            //if retro is not created
-//            if (retrofit == null) {
-//
-//                // create it
-//                retrofit = retrofit2.Retrofit.Builder()
-//                    .baseUrl(BASE_URL)
-//                    .addConverterFactory(GsonConverterFactory.create())
-//                    .build()
-//            }
-//
-//            return retrofit
-//        }
```

Comment:
cleared out the comments to create retro

Reply:
A good idea, as it's considered bad practice to commit commented out code to version control.  You can 1) go back in history to see what used to be there, or 2) use branches, instead of commenting out code.